src/endpoints/characters.js

```diff
@@ -688,6 +688,7 @@ function convertWorldInfoToCharacterBook(name, entries) {
                 useProbability: entry.useProbability ?? false,
                 depth: entry.depth ?? 4,
                 selectiveLogic: entry.selectiveLogic ?? 0,
+                outletName: entry.outletName ?? '',
```

Comment:
The code that uses setWIOriginalDataValue uses `extensions.outlet_name`, so it should be reflected here as well.

```suggestion
                outlet_name: entry.outletName ?? '',
```